Sources/Support/DebugUI/DebugContentViews.swift

```diff
@@ -159,9 +160,26 @@ private struct DebugSummaryView: View {
 @available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
 private struct DebugOfferingView: View {
 
+    @State private var showingSubscriptionSheet = false
+    @State private var showingStoreSheet = false
```

Comment:
I wonder if this could be an enum (since we can only show one of the 2 states or none right?)

Reply:
`SwiftUI` could handle having more than one sheet :)

Reply:
Oh interesting! That's fine then